package/MDAnalysis/analysis/rms.py

```diff
@@ -138,70 +137,89 @@
 logger = logging.getLogger('MDAnalysis.analysis.rmsd')
 
 
-def rmsd(a, b, weights=None, center=False):
+def rmsd(a, b, weights=None, center=False, superposition=False):
     """Returns RMSD between two coordinate sets *a* and *b*.
 
     *a* and *b* are arrays of the coordinates of N atoms of shape N*3
```

Comment:
Instead of *asterisked*, these should be `backticked` since they're parameter names. See http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_numpy.html